tidb/src/main/java/io/tidb/bigdata/tidb/ClientSession.java

```diff
@@ -469,14 +469,13 @@ public List<String> getPrimaryKeyColumns(String databaseName, String tableName)
         .collect(Collectors.toList());
   }
 
-  public List<String> getUniqueKeyColumns(String databaseName, String tableName) {
```

Comment:
I have a question for this previous implementation, what if we have a table  like 
```
 Table |                        Create Table
-------+-------------------------------------------------------------
 test  | CREATE TABLE "test" (                                      +
       |   "a" int(11) DEFAULT NULL,                                +
       |   "b" int(11) DEFAULT NULL,                                +
       |   "c" int(11) DEFAULT NULL,                                +
       |   UNIQUE KEY "ab" ("a","b"),                               +
       |   UNIQUE KEY "bc" ("b","c")                                +
       | ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin
```

Reply:
There are two categories of scenarios using this method.
One is in Flink, which uses a set to deduplicate
https://github.com/tidb-incubator/TiBigData/blob/master/flink/flink-1.14/src/main/java/io/tidb/bigdata/flink/connector/TiDBDynamicTableFactory.java#L138-L158
The other is in presto/trino. The duplicate column won't affect the result of containsAll() and the create table SQL(`create table ce (c1 bigint not null, c2 bigint not null, c3 bigint, c4 bigint, unique key(c1), unique key(c1));`)
https://github.com/tidb-incubator/TiBigData/blob/master/trino/src/main/java/io/tidb/bigdata/trino/tidb/TiDBMetadata.java#L202-L218

Reply:
Change the return value of `getUniqueKeyColumns` seems cause some code churn. The only place need the new return value is `dynamicTableFactory`. How about keep the current impl(maybe add the deduplication since the current seems a little bit error prone) and get the information in `dynamicTableFactory` only? Or introduce a `getUniqueKeys` method and do some extraction at the caller?